cfgov/unprocessed/css/molecules/expandable.less

```diff
@@ -0,0 +1,272 @@
+
+/* topdoc
+  name: Theme variables
+  family: cf-molecules
+  notes:
+    - "The following color and sizing variables are exposed,
+       allowing you to easily override them before compiling."
+  patterns:
+  - name: Colors
+    codenotes:
+      - |
+        @expandable_label-text
+        @expandable_link-text
+  - name: Sizing
+    codenotes:
+      - |
+        @expandable_label-font-size
+        @expandable_link-font-size
+  tags:
+  - cf-molecules
+  - less
+*/
+
+// Color variables
+// $color- variables are from 18F's US Web Design Standards
+// https://github.com/18F/web-design-standards/blob/18f-pages/assets/_scss/core/_variables.scss
+
+// .expandable_label
+@expandable_label-text:         #000000;
+
+// .expandable_link
+@expandable_link-text:          #046b99; // $color-primary-alt-darkest
+
+// Sizing variables
+
+// .expandable_link
+@expandable_label-font-size:    16px;
```

Comment:
Yeah, I'd point it at the base var so that if that changes, it changes here as well.